librz/include/rz_debug.h

```diff
@@ -410,6 +410,7 @@ typedef struct rz_debug_pid_t {
 	int uid;
 	int gid;
 	ut64 pc;
+	ut64 tls;
```

Comment:
add a doxygen comment here explaining what is `tls` (like `///< my comment`)

Reply:
Updated